virtualization/windowscontainers/manage-containers/walkthrough-iis-serviceaccount.md

```diff
@@ -0,0 +1,370 @@
+## Create a resource group
+Create a resource group in Azure to hold all of the resources you'll be creating:
+
+- Virtual Network
+- Virtual Machines
+- 
+
+## Create a VNet
+
+- Don't use 172.* IPs. I used 10.3.0.0/24
+
+## Deploy Azure VMs
+
+Deploy 3x VMs using the "Windows Server 2016 Datacenter - with Containers" image from marketplace - "dc", "Host1", "Host2"
+
+- Place them in the resource group above
+- Use the existing VNet
+- Assign a public IP
+- Use a Network Security Group with
+ - Inbound Rules
+  - RDP (TCP/3389) allowed
+  - Everything else blocked
+ - Outbound Rules
+  - Nothing 
+
+
+> Security Badness: Don't give your domain controller RDP access with a public IP
+
+
+## Start a domain
+
+Connect to dc
+
+1. ~~Assign a static IP address~~ - Azure already does this for you. Take note of it eg: `10.3.0.4`
+2. Install role & create domain
+```powershell
+install-windowsfeature AD-Domain-Services
+Import-Module ADDSDeployment
+Install-ADDSForest -DomainName contoso.local -DomainNetbiosName contoso -CreateDnsDelegation:$false -DatabasePath "C:\Windows\NTDS" -DomainMode "Win2012R2" -ForestMode "Win2012R2" -InstallDns:$true -LogPath "C:\Windows\NTDS" -NoRebootOnCompletion:$false -SysvolPath "C:\Windows\SYSVOL" -Force:$true
+# ^^ will prompt for safeadminpassword
+# This next step is bad. Don't do it in production. I'm not a Kerveros expert so I won't try to explain why
```

Comment:
It's definitely optional. Per some other feedback I'll probably split this into multiple sections but use the same names throughout the examples. My goal is that you can copy & paste line by line and have it work in clean VMs.

- set up a test environment - VMs & domain controller
- Set up a GMSA and create a corresponding credentialspec
- Create your first container using it to check end to end connectivity
- Deploy a workload
  - Create a IIS web site with user auth
  - IIS with service to service auth to SQL